package.json

```diff
@@ -23,14 +23,14 @@
   },
   "homepage": "https://github.com/vieron/stylelint-webpack-plugin#readme",
   "peerDependencies": {
-    "stylelint": "^7.0.1"
+    "stylelint": "^7.0.1",
+    "webpack": "^2 || ^2.2.0-rc.0 || ^2.1.0-beta.0 || ^1.13.2"
```

Comment:
There are no breaking changes between webpack 1 and webpack 2's API that would affect this plugin, it's very safe to support the beta versions of webpack, and gets rid of the annoying peer dependency warning on install.